README.md

```diff
@@ -87,7 +90,7 @@ NAME   TYPE        PATHNAME   AGE
 ns     Namespace   ch-ns      20s
 ```
 
-Resouces got moved to it destination, and `subscription` can take over from here, deploying this resource(it will be the configmap deployed) to your managed cluster. 
+Resources got moved to it destination, and `subscription` can take over from here, deploying this resource (it will be the configmap deployed) to your managed cluster. 
 
```

Comment:
I tried with line 93, but hard to understand. Needs some work but I was unable to rewrite it with confidence, can you please look into this line?

Reply:
Resources got moved to their destination managed/spoke cluster namespace. Then the `subscription` controller takes over from there. The `subscription` controller will deploy resources (in this case, it will be the configmap deployed) to your managed cluster.